assessment/exceptions/exceptions.py

```diff
@@ -8,3 +8,11 @@ class AssessmentToolDoesNotExist(ObjectDoesNotExist):
 
 class InvalidTestFlowRegistration(InvalidRequestException):
     pass
+
+
+class InvalidAssessmentEventRegistration(InvalidRequestException):
```

Comment:
Hello, Jo. If I may ask, will you be returning a custom error message for these errors?

Reply:
Hello, Alex. Yes, we will be returning custom error message through the service functions.

Reply:
Okay jo. Thanks for the update.